ch03.asciidoc

```diff
@@ -286,6 +286,10 @@ In that case the fees of the commitment transaction are again paid by the partne
 Also all the routing attempts will have to be resolved just as during the force close.
 While this method could be fully executed faster than the good and the bad way to close the channel, it is obviously never recommended to engage in this channel closing protocol.
 
+So, how do you catch a cheat or a protocol breach in your day-to-day activities? You do so by running software that monitors the public Bitcoin blockchain for on-line transactions that correspond to any of your commitment transaction. A Lightning node can do this monitoring for you. Specific Lightning wallets might do this for you. Running a Lightning node 24x7 will catch all cheats. But even if you run your Lightning node only once every Sunday, it will still catch all cheats because the timelock is roughly two weeks. In addition, you can delegate this responsibility to someone else. Someone else can monitor the Bitcoin blockchain for you. One of these technologies is called a "Watchtower".
```

Comment:
we should emphasze that the two weeks are only true if the lightning network was configured in this way and channels have been set up with long enough time locks. in general this statement does not need to be true

Reply:
OK, agreed. I added more text to make the risks more clear and give enough warning.
The new changes are under Commit #2 under this PR 
commit 232a81a